src/Functions/FunctionsStringBytes.cpp

```diff
@@ -0,0 +1,142 @@
+#include <cmath>
```

Comment:
We prefer every function in a separate file, as it opens possibilities to, e.g., link to a file from the docs, provide more precise authorship info, etc. So let's split into two files.

And then the .h file will be unneeded, so we can keep everything inside two .cpp files.

These .cpp files should be named exactly as the function, e.g., `stringBytesUniq.cpp`

Reply:
I understand it will be good to separate the core logic of the function in individual .cpp files, but wouldn't we still need a common .h file to re-use the `class FunctionStringBytes` for processing? Wouldn't re-implementing them for individual program lead to code repetition? 
Sorry if I misunderstood something.

Reply:
Yes, you are right. Let's keep FunctionStringBytes in the header file. The .cpp files will contain the corresponding Impl, instantiate the template, and register the function.

Reply:
Made this change as well.